CONTRIBUTING.rst

```diff
@@ -49,34 +56,81 @@ remember that SimPEG is maintained on a volunteer basis. If you suggest an
+
+.. _documentation: https://docs.simpeg.xyz/content/basic/practices.html#documentation
+
+.. _code style: https://docs.simpeg.xyz/content/basic/practices.html#style
+
 If you have an idea for how to improve SimPEG, please first create an issue
-and :ref:`suggest an enhancement <suggesting_enhancements>`. We will use the
+and `suggesting_enhancements`_. We will use the
 issue as a place to discuss and make decisions on the suggestion. Once you are
 ready to take action and commit some code to SimPEG, please check out
-:ref:`Getting Started for Developers <getting_started_developers>` for
-tips on setting up a development environment and :ref:`Practices <practices>`
+`getting started`_ for
+tips on setting up a development environment and `practices`_
 for a description of the development practices we aim to follow. In particular,
 
-- :ref:`testing <testing>`
-- :ref:`documentation <documentation>`
-- :ref:`code style <style>`
+- `testing`_
+- `documentation`_
+- `code style`_
 
 are aspects we look for in all pull requests. We do code reviews on pull
 requests, with the aim of promoting best practices and ensuring that new
 contributions can be built upon by the SimPEG community.
 
+.. _pr_stages:
+
+Stages of a pull request
+------------------------
+
+When first creating a pull request, try to make your suggested changes as tightly
+scoped as possible. The fewer changes you make, the faster your branch will be
+merged!
+
+If your pull request is not ready for final review, but you still want feedback
+on your current coding process please mark it as a draft pull request. Once you
+feel the pull request is ready for final review, you can convert the draft PR to
+an open PR by selecting the ``Ready for review`` button at the bottom of the page.
+
+Once a pull request is in ``open`` status and you are ready for review, please ping
+the simpeg developers in a github comment ``@simpeg/simpeg-developers`` to request a
+review. At minimum for a PR to be eligible to merge, we look for
+
+- 100% (or as close as possible) difference testing. Meaning any new code is completely tested.
+- All tests are passing.
+- All reviewer comments (if any) have been addressed.
+- A developer approves the PR.
+
+After all these steps are satisfied, a ``@simpeg/simpeg-admin`` will merge your pull request into
+the main branch (feel free to ping one of us on github).
+
+This being said, all simpeg developers and admins are essentially volunteers
+providing their time for the benefit of the community. This does mean that
+it might take some time for us to get your PR.
+
 
 Licensing
-*********
+=========
 
-The contributed code will be licensed under SimPEG's license
-(https://github.com/simpeg/simpeg/blob/main/LICENSE). If you did not write
+The code contributed to SimPEG will be licensed under the `MIT license
+<https://github.com/simpeg/simpeg/blob/main/LICENSE>`_ which allows open
+and commercial use and extension of SimPEG. If you did not write
```

Comment:
I think it would be nice to make it clearer **when** the code contributed to SimPEG becomes licensed under the MIT License. The _will_ might lead to think that this code will _eventually_ be MIT licensed, but without giving a clear time in the future.

In my opinion, once any contributor opens a PR, that code is already being shared under the MIT license. It's the author that licenses their contribution under that license, and not the package maintainers by merging their code.

```suggestion
All code contributed to SimPEG is licensed under the `MIT license
<https://github.com/simpeg/simpeg/blob/main/LICENSE>`_ which allows open
and commercial use and extension of SimPEG. If you did not write
```

Reply:
That's a good point, the language is quite subtle in general here.